### ISSUE TITLE
github integration: stop working

### ISSUE THREAD

**Issue by brownman** [GitbookIO/gitbook#508]
required:
the user need some indication regarding the gitbook-github integration status.

currently:
new commits on github doesn't trigger the gitbook compiler.

**Comment by kedych** [comment]
I got the same problem.
This is my gitbook: https://www.gitbook.com/read/book/ndc/newtaiwan
I have integrated with this: https://github.com/white-book/newtaiwan

I can see the content in both editor and github, however, I edit the book, click save, nothing build. 

I am sure that the modified content is sync with github. I can see the commit history at github.

The book shows the original content. In the "book history" shows nothing build.
Could you please tell me how to solve the problem? 

Best regards,

**Comment by xabier** [comment]
I have the same problem with 
https://www.gitbook.com/book/xabier/comunes-conocimiento-democracia/details
and

The gitbook editor does show updates from github, but gitbook doesn't build, not sure how to fix the webhook issue (claimed to be the main sources of problems on this matter), because I don't know what the correct webhook payload url is.

**Comment by SamyPesse** [comment]
Do you have a gitbook.com webhook listed at https://github.com/xabier/comunes-conocimiento-democracia/settings/hooks ?

If no, try adding one by clicking the button "Add a deployment webhook" at https://www.gitbook.com/book/xabier/comunes-conocimiento-democracia/settings/github

**Comment by dji-dev** [comment]
Hi SamyPesse,  I got the same problem.

This is my gitbook: https://www.gitbook.com/book/dji-dev/mobile-sdk-tutorials/details
I have integrated with this: https://github.com/dji-sdk/Mobile-SDK-Tutorial

I can only see the content updated in the Github, and I also try to edit the book, click save, nothing build.

I am sure that the modified content is sync with Github. I can see the commit history at github.

The book shows the original content. In the "book history" shows nothing build.

Finally, I check the Webhooks&services settings in my Github project, and in the Recent Deliveries, I can see the Response 500 error, the body is :

Could you please tell me how to solve the problem?

Best regards,

**Comment by xabier** [comment]
My problem disappeared, but I tried far too many changes to be capable to know which change actually had the right effect. Sorry for that, I will try to be more systematic next time to identify where the problem lies.

**Comment by kwiksher** [comment]
I am having the same problem Request Entity Too Large. It always appear when pushing and webhooks in github. From gitbook, adding webhook to github succeeded but pushing fires a hook but fails.

**Comment by SamyPesse** [comment]
@kwiksher we have a limit on the book size that can be pulled during a webhook; how big is your book folder ?

**Comment by kwiksher** [comment]
the size of_book folder is about 125mega bytes

**Comment by xpgeng** [comment]
Hi SamyPesse,
I got the same problem like @xabier .
- I have checked the webhook, There isn't ant webhook in the list.
- I click the botton"add a deployment webhook" , it only shows "Book settings has been saved! " 
- I also  return to github to add the webhook by hand, I use the Payload url "<https://git.gitbook.com/xpgeng/omooc2py.git >". But it showed that "Invalid HTTP response:0".

So, what can I do next to publish my book linking to github?

Thanks very much

Best regards,
